app/src/main/kotlin/com/simplemobiletools/clock/activities/MainActivity.kt

```diff
@@ -8,33 +8,34 @@ import android.graphics.drawable.Icon
 import android.graphics.drawable.LayerDrawable
 import android.os.Bundle
 import android.view.WindowManager
-import android.widget.ImageView
-import android.widget.TextView
 import com.simplemobiletools.clock.BuildConfig
 import com.simplemobiletools.clock.R
 import com.simplemobiletools.clock.adapters.ViewPagerAdapter
+import com.simplemobiletools.clock.databinding.ActivityMainBinding
 import com.simplemobiletools.clock.extensions.*
 import com.simplemobiletools.clock.helpers.*
+import com.simplemobiletools.commons.databinding.BottomTablayoutItemBinding
 import com.simplemobiletools.commons.extensions.*
 import com.simplemobiletools.commons.helpers.*
 import com.simplemobiletools.commons.models.FAQItem
-import kotlinx.android.synthetic.main.activity_main.*
 import me.grantland.widget.AutofitHelper
 
 class MainActivity : SimpleActivity() {
     private var storedTextColor = 0
     private var storedBackgroundColor = 0
     private var storedPrimaryColor = 0
+    private lateinit var binding: ActivityMainBinding
```

Comment:
https://github.com/SimpleMobileTools/Simple-Commons/blob/0bdbc422dd71e67e0264b01e72799448cc5c23f2/commons/src/main/kotlin/com/simplemobiletools/commons/activities/LicenseActivity.kt#L17

Reply:
Forgot about that, thanks :smile: